x-pack/plugins/console_extensions/spec/overrides/xpack.ml.put_datafeed.json

```diff
@@ -0,0 +1,16 @@
+{
+  "xpack.ml.put_datafeed": {
+    "data_autocomplete_rules": {
+      "aggregations": {},
+      "chunking_config": {},
+      "frequency": "",
+      "indices": [""],
```

Comment:
This array is not pre-populated with an empty string; it's inserted as an empty array.

Reply:
fixed